sceptre/stack.py

```diff
@@ -120,10 +120,11 @@ class Stack(object):
     hooks = HookProperty("hooks")
 
     def __init__(
-        self, name, project_code, region, template_path=None, template_handler_config=None,
-        template_bucket_name=None, template_key_prefix=None, required_version=None,
-        parameters=None, sceptre_user_data=None, hooks=None, s3_details=None,
-        iam_role=None, dependencies=None, role_arn=None, protected=False, tags=None,
+        self, name, rel_path, project_code, region, template_path=None,
```

Comment:
Same comment above, `rel_path` is not a good variable name. Let's not abbreviate like that.

Beyond that, though, I'm wondering _why_ this is necessary? What _is_ "rel_path"? What purpose does it serve? There's no documentation on that parameter here. It's hard for me to determine what is going on here.